pkg/cluster/cluster.go

```diff
@@ -305,7 +305,7 @@ func CreateCluster(ctx context.Context, cluster *k3d.Cluster, runtime k3drt.Runt
 			}
 			cluster.Nodes = append(cluster.Nodes, lbNode) // append lbNode to list of cluster nodes, so it will be considered during rollback
 			log.Infof("Creating LoadBalancer '%s'", lbNode.Name)
-			if err := CreateNode(lbNode, runtime); err != nil {
```

Comment:
@ibuildthecloud somehow this is always failing like this:
```
DEBU[0016] FIXME: Got an status-code for which error does not match any expected type!!!: -1  module=api status_code=-1    # <- this is from docker
ERRO[0016] Failed to create container 'k3d-demo-loadbalancer-0' 
...       
ERRO[0016] error during connect: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create?name=k3d-demo-loadbalancer-0": context canceled
```
I'm still fairly new to using context.Context.. do you have a clue, why the context gets canceled early here? :thinking:

Reply:
fixed in d3766a5